src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4680,4 +4680,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PeWritingFailure" xml:space="preserve">
     <value>An error occurred while writing the output file: {0}</value>
   </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning" xml:space="preserve">
+    <value>Attribute '{0}' is ignored when public signing is specified.</value>
+  </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning_Title" xml:space="preserve">
```

Comment:
We display only non-error diagnostics as these are the only ones whose effective severity can be configured by the end user (compilation options).